CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#1933](https://github.com/thanos-io/thanos/pull/1933) Add a flag `--tsdb.wal-compression` to configure whether to enable tsdb wal compression in ruler and receiver.
 - [#2021](https://github.com/thanos-io/thanos/pull/2021) Rename metric `thanos_query_duplicated_store_address` to `thanos_query_duplicated_store_addresses_total` and `thanos_rule_duplicated_query_address` to `thanos_rule_duplicated_query_addresses_total`.
+- [#2048](https://github.com/thanos-io/thanos/pull/2048) Improve help text of `thanos_shipper_dir_syncs_total`, `thanos_shipper_uploads_total` and `thanos_shipper_upload_failures_total` metrics.
```

Comment:
No need to report this here (: No user is impacted, it's just documentation change. No logic was changed.